### PR TITLE
trigger_openshift_ci_job - start job

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

File: app/app.py [hunk 1]
```diff
@@ -59,8 +65,20 @@ def process_hook(api, data):
 
 
 def trigger_openshift_ci_job(job):
-    # TODO: Call openshift-ci to trigger the job
+    openshift_ci_token_env_variable_str = "OPENSHIFT_CI_TRIGGER_TOKEN"
```
Comment (myakove) [inline app/app.py:68]: Token can be either os env or configured in the yaml

File: example.config.yaml [hunk 2]
```diff
@@ -3,3 +3,5 @@ repositories:
     webhook_ip: <HTTP://IP OR URL:PORT>
     name: RedHatQE/openshift-python-wrapper
     project_id: <PROJECT ID>
+
+trigger_url: <url to send the trigger request to>
```
Comment (myakove) [inline example.config.yaml:7]: It always the same trigger url? and if so please put it in the top of the file

File: app/app.py [hunk 3]
```diff
@@ -59,8 +65,20 @@ def process_hook(api, data):
...
     app.logger.info(f"Triggering openshift-ci job: {job}")
+    token = os.environ.get(openshift_ci_token_env_variable_str)
+    assert token, f"{openshift_ci_token_env_variable_str} is not set"
+    res = requests.post(
+        url=f"{data_from_config()['trigger_url']}/{job}",
+        headers={"Authorization": f"Bearer {token}"},
+        data='{"job_execution_type": "1"}',
+    )
+    res_dict = json.loads(res.text)
```
Comment (myakove) [inline app/app.py:77]: do we want to return `res_dict` ?

**Review by myakove — CHANGES_REQUESTED** [review]

File: app/app.py (hunk 3 above)
Comment (rnetser) [reply app/app.py:77]: done

File: app/app.py (hunk 1 above)
Comment (rnetser) [reply app/app.py:68]: moved it to the config

**Review by myakove — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Pull request merged**